fetch.bs

```diff
@@ -1410,9 +1413,11 @@ Unless stated otherwise, it is unset.
  <dl>
   <dt>"<code>default</code>"
   <dd><a for=/>Fetch</a> will inspect the HTTP cache on the way to the network.
-  If there is a fresh response it will be used. If there is a stale response a conditional request
-  will be created, and a normal request otherwise. It then updates the HTTP cache with the response.
-  [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]]
+  If there is a fresh response it will be used. If there is a stale response it will be returned immediately
+  if the request <a for=request>allows stale content flag</a> is set. A conditional request will be created
```

Comment:
I feel like I'm missing something, but isn't this going to cause most requests to return stale content? It looks like "allows stale content flag" will be frequently set, as long as it "is a request that is a  subresource request and that does not have the unsafe-request flag set. "